### ISSUE TITLE
Demonstrate how to secure an endpoint

### ISSUE THREAD

**Issue by gala-dan** [GalateaRaj/fuse-starter-java#300]
Issue Description
--
Would be useful to demonstrate how to secure a restful endpoint to ensure only authenticated users can access it.

Design
--
[Description of the root cause for the issue and how to resolve it. Should be complete before Review is approved]

Documentation Changes
--
[Description of changes necessary to the Fuse documentation due to the changes in response to this issue. Should be complete before Review is approved]

Test Evidence
--
[include relevant tests and output. Should be complete before Review is approved]

Validation in Develop
-- 
[Proof issue is resolved in Develop. Should be complete before Issue is closed]

**Comment by GalateaRaj** [comment]
Duplicate of #33
